Locale/frFR/config.lua

```diff
@@ -2,180 +2,180 @@ QuestieLocale.locale['frFR'] = {
 
 	['ADV_TAB'] = "Avancé",
-	['DEV_OPTIONS'] = "Options du développeur",
-	['ENABLE_DEBUG'] = "Activer le debug",
-	['ENABLE_DEBUG_DESC'] = "Activer ou désactiver la fonctionnalité de debug.",
-	['DEBUG_LEVEL'] = "Niveau de debug à imprimer",
-	['DEBUG_LEVEL_DESC'] = "Quel niveau de debug pour imprimer à: %s",
-	['LOCALE'] = "Paramètres de localisation",
-	['LOCALE_DROP'] = "Sélectionnez l'interface utilisateur",
-	['RESET_QUESTIE'] = "Réinitialiser Questie",
-	['RESET_QUESTIE_DESC'] = "Si vous cliquez sur ce bouton, tous les paramètres de configuration de Questie seront réinitialisés à leurs valeurs par défaut. (Hors localisation / Traduction)",
+	['DEV_OPTIONS'] = "Options de développement",
+	['ENABLE_DEBUG'] = "Mode debug",
+	['ENABLE_DEBUG_DESC'] = "Active le mode debug.",
+	['DEBUG_LEVEL'] = "Niveau de debug",
+	['DEBUG_LEVEL_DESC'] = "Définis le niveau du mode debug : %s",
+	['LOCALE'] = "Paramètre de localisation",
+	['LOCALE_DROP'] = "Sélection de la langue :",
+	['RESET_QUESTIE'] = "Réinitialisation",
+	['RESET_QUESTIE_DESC'] = "Activer ce bouton réinitialisera tous les paramètres de Questie à leurs valeurs par défaut. (sauf choix de la langue)",
 	['RESET_QUESTIE_BTN'] = "Réinitialiser Questie",
 	['RESET_QUESTIE_BTN_DESC'] = "Réinitialiser Questie aux valeurs par défaut pour tous les paramètres.",
-	['QUESTIE_DEV_MESSAGE'] = "Questie est en cours de développement pour World of Warcraft: Classic. Veuillez consulter GitHub pour connaître les dernières versions alpha ou signaler des problèmes. Ou rejoignez nous sur notre discord! (( https://github.com/AeroScripts/QuestieDev/ )) ",
+	['QUESTIE_DEV_MESSAGE'] = "Questie est en cours de développement pour World of Warcraft: Classic. Veuillez consulter GitHub pour connaître les dernières versions alpha ou signaler des problèmes. Ou rejoignez nous sur notre discord ! ((https://github.com/AeroScripts/QuestieDev/)) ",
 	
 	-- Éléments d'interface utilisateur
 	['QUESTIE_MAP_BUTTON_SHOW'] = "Afficher Questie",
 	['QUESTIE_MAP_BUTTON_HIDE'] = "Masquer Questie",
-	['ICON_LEFT_CLICK'] = "Clic Gauche",
-	['ICON_TOGGLE'] = "Options de bascule",
-	['ICON_SHIFTLEFT_CLICK'] = "Maj + Clic Gauche",
-	['ICON_TOGGLE_QUESTIE'] = "Basculer la Questie",
-	['ICON_CTRLRIGHT_CLICK'] = "Ctrl + Clic Droit",
-	['ICON_CTRLLEFT_CLICK'] = "Ctrl + Clic Gauche",
-	['ICON_HIDE'] = "Bouton Masquer la mini-carte",
-	['ICON_RIGHT_CLICK'] = "Clic Droit",
-	['ICON_JOURNEY'] = "Basculer mon voyage",
+	['ICON_LEFT_CLICK'] = "Clic Gauche ",
+	['ICON_TOGGLE'] = "Panneau de configuration",
+	['ICON_SHIFTLEFT_CLICK'] = "Maj + Clic Gauche ",
+	['ICON_TOGGLE_QUESTIE'] = "Afficher/cacher Questie",
+	['ICON_CTRLRIGHT_CLICK'] = "Ctrl + Clic Droit ",
+	['ICON_CTRLLEFT_CLICK'] = "Ctrl + Clic Gauche ",
+	['ICON_HIDE'] = "Cacher le bouton autour de la mini-carte",
+	['ICON_RIGHT_CLICK'] = "Clic Droit ",
```

Comment:
```suggestion
	['ICON_RIGHT_CLICK'] = "Clic Droit",
```

Reply:
I added whitespaces in these ones so it follow the french whitespace rule for the " : " sign. This way the whole sentence looks like "Clic Gauche : Panneau de configuration" with "Clic Gauche" being displayed in white/grey and " : Panneau de configuration" in yellow.